app/models/organisation.rb

```diff
@@ -89,6 +89,10 @@ def self.create_and_validate(attributes)
     create!(attributes.select{|k,v| !v.nil?})
   end
 
+  def self.create_and_substitute_with_empty(attributes)
+    create!(attributes.map { |k, v| attributes[k] = v.nil? ? ' ' : v })
```

Comment:
note you are turning a hash into an array here, which I don't think is what you want ... see http://stackoverflow.com/questions/5189161/changing-every-value-in-a-hash-in-ruby perhaps